### PR TITLE
Yogi

### PR DESCRIPTION
**Pull request by YOG3SHWAR**
Update profile screen is complete, review and merge it.

### CODE REVIEW

File: html/profile.html [hunk 1]
```diff
@@ -10,9 +10,91 @@
 </head>
 
 <body>
-  <h1 class="title-of-page">Setting up Profile.</h1>
 
-  <script src="../script/profile.js"></script>
+  <div class="app">
```
Comment (shubhamch-er) [inline html/profile.html]: class name can be `container` or `profile-container` ?

File: html/profile.html [hunk 2]
```diff
@@ -10,9 +10,91 @@
...
+
+    <div class="app__body">
+
+      <div class="sidebar">
+
+        <div class="sidebar__header">
+          <img src="../image/img_avatar.png" alt="avtar" class="avatar">
```
Comment (shubhamch-er) [inline html/profile.html]: spelling of avatar is misspelled in `alt` attribute

File: html/profile.html [hunk 3]
```diff
@@ -10,9 +10,91 @@
...
+          <button class="button" style="vertical-align:middle"><span>Name</span>
```
Comment (shubhamch-er) [inline html/profile.html]: style should be in css file. Avoid inline styling.

File: html/profile.html [hunk 4]
```diff
@@ -10,9 +10,91 @@
...
+        </div>
+
+        <div class="sidebar__search">
+          <div class="sidebar__searchContainer">
+            <img src="../image/search.png" alt="search" class="search">
+            <input placeholder="Search" type="search">
+          </div>
+        </div>
+
+        <div class="sidebar__chat">
```
Comment (shubhamch-er) [inline html/profile.html]: `sidebar__friend-list` can be a good name i think ?

**Review by shubhamch-er — CHANGES_REQUESTED** [review]
Please look at the comments. And try to change wherever you feel improvement can be done.

File: html/profile.html (hunk 2 above)
Comment (YOG3SHWAR) [reply html/profile.html]: Corrected it

File: html/profile.html (hunk 3 above)
Comment (YOG3SHWAR) [reply html/profile.html]: Moved it to css file

File: html/profile.html (hunk 4 above)
Comment (YOG3SHWAR) [reply html/profile.html]: changed it to sidebar__friend-list

File: html/profile.html (hunk 1 above)
Comment (YOG3SHWAR) [reply html/profile.html]: It will be problematic because I have used class profile in that file, so it will be confusing.

Comment (shubhamch-er) [reply html/profile.html]: ok..

**Pull request merged**